README.md

```diff
@@ -71,6 +71,17 @@ For running in CI servers, we use Karma which works perfectly fine!
 
     - `entry` should be set to the test entry file. For example, `./test-entry.js`.
 
+    - optionally set `webpackMiddleware` which will be merged with some default options when we initialize webpack-middleware. This can be useful in case you need to enable polling:
+    ```
+      webpackMiddleware: {
```

Comment:
Do you think `devServer` or `webpackDevMiddleware` would be a better name choice?

- `devServer` is already used by `webpack-dev-server` which already includes `webpack-dev-middleware` options.

- `webpackDevMiddleware` on the other hand, makes it clearer because we use two webpack middlewares: `webpack-dev-middleware` and `webpack-hot-middleware`